src/containers/LoginRegister/index.jsx

```diff
@@ -0,0 +1,201 @@
+import React, { useContext, useState } from "react";
+import { withRouter } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import { Alert, Spin } from "antd";
+import firebase from "firebase";
+import Login from "../../components/userauth/Login";
+import Register from "../../components/userauth/Register";
+import ResetPassword from "../../components/userauth/ResetPassword";
+import firestore, { auth } from "../../firebaseConfig";
+import "./index.css";
+import kids from "../../images/LoginKids.svg";
+
+const LoginRegisterPage = ({ history }) => {
+  const [t] = useTranslation();
+  const [active, setActive] = useState("login");
+  const [error, setError] = useState("");
+  const [message, setMessage] = useState();
```

Comment:
Since message is defined as string Reset password, I think you may add double quotes as;
 `useState("");`

Reply:
"" is equal to undefined too. But still I applied what you suggested.

Reply:
what do you mean "" is equal to undefined? i would say not quite. your code will crash in a lot of scenarios if you dont make it "" (although making it undefined is also a decision you could make, but usually we prefer null to express values that are "not present")

for example...

if you have code that does message.trim(), it will crash if this is undefined

but basically what sule says is the way to go usually. it also gives readability by indicating this should be a string

Reply:
Makes sense. I usually do what you said to prevent errors. This time it slipped out of my attention. Thank you for pointing out @ssinikoglu and @louisrli